tests/main/ubuntu-core-refresh/task.yaml

```diff
@@ -9,6 +9,23 @@ details: |
 # TODO:UC20: enable for UC20
 systems: [ubuntu-core-1*]
 
+
+restore: |
+    #shellcheck source=tests/lib/systems.sh
+    . "$TESTSLIB"/systems.sh
+    TARGET_SNAP_NAME=core
+    if is_core18_system; then
+        TARGET_SNAP_NAME=core18
+    fi
+
+    # Wait for the system to stop processing stuff
+    # XXX: What was it processing? I got this error:
+    # error: snap "core" has "revert-snap" change in progress
+    while snap changes | grep Doing; do
```

Comment:
same comment about retry-tool, can we use that here instead?

Reply:
I think we could but it would involve running a shell, let me try.

Reply:
I changed this to use `snap watch --last=` instead. Please have a look.